sys/hashes/cmac.c

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2016 Fundación Inria Chile
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_hashes_cmac
+ * @{
+ *
+ * @file
+ * @brief       AES_CMAC implementation
+ *
+ * @author      José Ignacio Alamos <jose.alamos@inria.cl>
+ *
+ * @}
+ */
+
+#include <inttypes.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "crypto/ciphers.h"
+#include "hashes/cmac.h"
+
+#define MIN(a, b) a < b ? a : b
+
+static inline void _xor128(uint8_t *x, uint8_t *y)
+{
+    uint8_t i;
+    for (i = 0; i < 16; i++) {
+        y[i] = x[i] ^ y[i];
+    }
+}
+
+static inline void _leftshift(uint8_t *x, uint8_t *y)
+{
+    uint8_t i;
+    for (i = 0; i < 15; i++)
```

Comment:
please
- put variable initializer in for (...),
- use unsigned as variable,
- put opening bracket on same line